src/pois.rs

```diff
@@ -176,6 +320,22 @@ impl IndexedPoi {
             &country_codes,
             langs,
         );
+        let mut self_country_langs_to_add: Vec<String> = Vec::new();
+        for country_code in country_codes.iter() {
+            if let Some(langs) = COUNTRIES_LANGS.get(country_code) {
+                for lang in langs {
```

Comment:
Languages that are not included in the `langs` options should be excluded here.

Reply:
If we do so, the country in which this POI is in won't have a label with its language. For example, louvre doesn't have "fr" label and we if don't pass the language, it won't be added.